requirements.txt

```diff
@@ -4,3 +4,4 @@ kombu # A celery dependency but it's directly imported
 requests_kerberos
 requests
 semver
+pyyaml
```

Comment:
This implies that we need `pyyaml` to run Cachito itself. Can we just add this to `requirements-dev.txt` instead? Or maybe a new file, `requirements-integration-test.txt` ?

Reply:
Seems like reusing `requirements-dev.txt` might be a good idea since it already includes tox?

Reply:
I moved it to `requirements-dev.txt`